app/views/citizens/means_test_results/show.html.erb

```diff
@@ -1,22 +1,26 @@
 <% content_for(:panel) do %>
-  <div class="govuk-panel govuk-panel--confirmation">
-    <h1 class="govuk-panel__title govuk-!-padding-top-4">
-      <%= t(".completed_financial_assessment") %>
-    </h1>
-    <div class="govuk-panel__body">
-      <dl class="govuk-list inline-list">
-        <dt>
-          <strong><%= t(".name") %>:</strong>
+  <%= govuk_panel(
+    title_text: t(".completed_financial_assessment")) do %>
+    <dl class="govuk-summary-list govuk-summary-list--no-border govuk-list inline-list">
```

Comment:
There's a govuk_summary_list component, like the govuk_panel component. [Info here](https://govuk-components.netlify.app/components/summary-list/)

I've not tried using it within a panel but I'm assuming it'll work

Reply:
Thank you for the suggestion. Do you think it would be best if this was done as part of the summary_list component ticket(s)?